recipes/plasmidid/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin/config_files
+mkdir -p $PREFIX/config_files
+
+cp bin/* $PREFIX/bin
+cp plasmidID $PREFIX/bin
+
+cp config_files/* $PREFIX/bin/config_files
+
+echo "INSTALING MASH 2"
+echo "#############################################"
+
+wget https://github.com/marbl/Mash/releases/download/v2.2/mash-Linux64-v2.2.tar
+tar -xvf mash-Linux64-v2.2.tar
+cp mash-Linux64-v2.2/* $PREFIX/bin
+
+echo "INSTALING CIRCOS WITH CUSTOM SETTINGS"
+echo "#############################################"
+
+wget http://circos.ca/distribution/circos-0.69-8.tgz
+tar -zxvf circos-0.69-8.tgz
+cd circos-0.69-8
+sed -i "s/max_points_per_track =.*/max_points_per_track = 80000000/g" etc/housekeeping.conf
```

Comment:
I haven’t used Circos myself, only briefly looked at the tutorial right now. From that, I wonder whether you actually should modify the `etc/housekeeping.conf` file, no matter in which way you install plasmidID. Wouldn’t it be possible tojust add
```
max_points_per_track = 80000000
```
after each
```
<<include etc/housekeeping.conf>>
```
in the Circos configurations that are part of plasmidID?

Reply:
Thank you for the guidance.
-Mash now works by specifying conda-forge channel (I swear I tried already).
-Circos is now installed properly and the specific line in housekeeping.conf is modifyed in post-link.sh file. This line doesn't afect circos overal performance.

Reply:
Thanks for the changes. I’m not entirely sold, yet, on using `post-link.sh` to modify another package’s configuration. Perhaps a second opinion is necessary. @dpryan79, do you have time to give us your thoughts on this?

Reply:
Am I correct in understanding that the post-link script is modify a circos conf file? I'm really uncomfortable doing that, packages should really not modify each other.

Reply:
Yes, that’s what the script does. @pedroscampoy What about the suggestion to change plasmidID itself so that it sets this configuration value correctly? Or would that not work?

Reply:
I have adapted the code so I no longer have to modify any circos config file. Thank to your suggestions, they have been very helpful.
Please review the recipe whenever is convenient for you.